6_handbook/20_application_tuning.txt

```diff
@@ -208,3 +208,16 @@ mount /var/rudder/cfengine-community/state
 
 ----
 
+==== Rsyslog
+
+On Rudder policy servers (root or relay), when managing a large number of nodes,
+you can experience issues with rsyslog. This happens because Rudder uses TCP
```

Comment:
Right. But I would still like the text to read "uses TCP by default". Because we can always change it manually, even before the configurable option.